core/src/display_object/container.rs

```diff
@@ -289,6 +289,10 @@ pub trait TDisplayObjectContainer<'gc>:
     }
 
     /// Iterates over the children of this display object in depth order.
+    /// Unless you are performing an AVM1-specific operation, you should
```

Comment:
The only remaining usage of `iter_depth_list` is the Ctrl+Alt+V AVM1 `VariableDumper` in `player.rs`. I think it's a good opportunity to greatly simplify `iter_depth_list` (along with `DepthIter` and `lowest_depth`), maybe by iterating over an `i32` and call `child_by_depth` each time?

Reply:
I considered that, but I was concerned that we might need it again in the future. If you think it's a good idea, then I'll go ahead and remove it.

Reply:
I think `iter_depth_list` is not used not by chance - Flash probably doesn't have that functionality either. So I feel quite safe to remove it. Also, it's likely that `iter_children_by_depth` can fit as an alternative for the AVM1 `VariableDumper`.

Reply:
I used `iter_render_list` and manually retrieved the depth - under AVM1, the depth and render lists should be same.